src/render/Renderer.cpp

```diff
@@ -1489,6 +1483,17 @@ bool CHyprRenderer::commitPendingAndDoExplicitSync(PHLMONITOR pMonitor) {
         }
     }
 
+    static auto PWIDE    = CConfigValue<Hyprlang::INT>("experimental:wide_color_gamut");
+    const bool  needsWCG = *PWIDE || pMonitor->output->state->state().hdrMetadata.hdmi_metadata_type1.eotf == 2;
+    if (pMonitor->output->state->state().wideColorGamut != needsWCG) {
+        Debug::log(TRACE, "Setting wide color gamut {}", needsWCG ? "on" : "off");
+        pMonitor->output->state->setWideColorGamut(needsWCG);
+
+        // FIXME do not trust enabled10bit, auto switch to 10bit and back if needed
```

Comment:
why not pop a notification? (of course limit it to one per session or config load)

logs can be easily missed